examples/dashboard-with-property-dataset/dataset-dashboard.ts

```diff
@@ -0,0 +1,75 @@
+import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
+import { css, html, customElement, LitElement } from '@umbraco-cms/backoffice/external/lit';
+import { UmbElementMixin } from '@umbraco-cms/backoffice/element-api';
+import { UmbPropertyValueData, type UmbPropertyDatasetElement } from '@umbraco-cms/backoffice/property';
+
+@customElement('example-dataset-dashboard')
+export class ExampleDatasetDashboard extends UmbElementMixin(LitElement) {
+	data: UmbPropertyValueData[] = [
+		{
+			alias: 'textProperty',
+			value: 'Hello',
+		},
+	];
+
+	#onDataChange(e: Event) {
+		const oldValue = this.data;
+		this.data = (e.target as UmbPropertyDatasetElement).value;
+		this.requestUpdate('data', oldValue);
+	}
+
+	render() {
+		return html`
+			<uui-box class="uui-text">
+				<h1 class="uui-h2" style="margin-top: var(--uui-size-layout-1);">Dataset Example</h1>
+				<umb-property-dataset .value=${this.data} @change=${this.#onDataChange}>
+					<umb-property
```

Comment:
Would it be possible to set a value prop on the property element so it could work without the dataset context? Or how much chaos would that create? :D

Reply:
I know it is not the purpose of the example. It would be to "simplify" the simple use case of a property element.